Examples/Scripts/Python/full_chain_itk_FTF.py

```diff
@@ -52,7 +52,7 @@
             mean=acts.Vector4(0, 0, 0, 0),
         ),
         rnd=rnd,
-        outputDirRoot=outputDir,
+        # outputDirRoot=outputDir,
```

Comment:
same here and in `events=1000`. Are these intende to be merged?

Reply:
no sorry, thank you for catching this, will change now